demo/src/js/demo.js

```diff
@@ -212,6 +214,9 @@ function initializeWall () {
       const secondsFromStart = Math.floor(moment.duration(Date.now() - now).asSeconds()) - (hoursFromStart * 60 * 60) - (minutesFromStart * 60);
 
       $('#wallDuration').text(`${hoursFromStart} hours ${minutesFromStart} minutes ${secondsFromStart} seconds`);
+      $('#wallHeapSizeLimit').text(humanize.filesize(window.performance.memory.jsHeapSizeLimit));
```

Comment:
I'm presuming that the demo page is for Chrome only? If not, you should have a null check for window.performance.memory prior to use.